setup.py

```diff
@@ -20,7 +20,18 @@
 import logging
 import os
 
-import setuptools
+import setuptools, subprocess
+from setuptools.command.install import install
+from distutils.sysconfig import get_python_lib
+
+LIMINAL_HOME_DAG="/opt/airflow/dags/liminal_dags.py"
```

Comment:
We should get dags dir from airflow, airflow users can config a dags dir.

Reply:
The only indication about the dags dir can come from the requirements that we installed like [apache-airflow](https://github.com/apache/incubator-liminal/blob/master/requirements.txt#L20)
and even then the post installation is happening before the requirements.
 
WDYT?

Reply:
[The post-installation isn't possible](https://github.com/pypa/setuptools/issues/1782#issuecomment-500840482)
I should find another solution.
I can define extra path to DAGS_FOLDER and it will be in the side of the DevOps when he installs Airflow. Something like:
**AIRFLOW__CORE__DAGS_FOLDER=/opt/airflow/dags:/home/airflow/.local/lib/python3.6/site-packages/liminal/runners/airflow/dag**

WDYT?

I think it also won't work (multiple dags folder). I can change the **AIRFLOW__CORE__DAGS_FOLDER** to where liminal installs its dependencies and there the liminal_dags.py will take care to generate/read dags from the /opt/airflow/dags but then I interrupt the home folder of dags and not reading it from the env.

Reply:
My above comment isn't relevant.
I solved it in another way.